### ISSUE TITLE
Couldn't connect to the k8s cluster

### ISSUE THREAD

**Issue by FranciscoMSM** [yolossn/lazykubernetes#26]
I'm running lazykubernetes as:
`KUBECONFIG=PATH_TO_MY_KUBECONFIG lazykubernetes` and getting: `2020/09/15 11:55:01 Couldn't connect to the k8s cluster`

Can you please guide me on how to setup authentication?
Host: `Linux fmagalhaes-1 4.17.0-3-amd64 #1 SMP Debian 4.17.17-1 (2018-08-18) x86_64 GNU/Linux`

I'm also interested in helping into the project, if you can link me to the tickets with my priority I may have time to help you out!

Thanks for this initial effort!

**Comment by FranciscoMSM** [comment]
Ok I just read the client code:
```
	var kubeconfig string
	if home := homedir.HomeDir(); home != "" {
		kubeconfig = filepath.Join(home, ".kube", "config")
	}
	// } else {
	// 	kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
	// }
```

Would you mind if I add support to listen to the $KUBECONFIG env var?

**Comment by yolossn** [comment]
Yes! go ahead and raise a PR. We can also support `--kubeconfig` flag to specify the file also.
